steamcmd_servers/valheim/valheimplus/egg-valheim-plus.json

```diff
@@ -0,0 +1,113 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-03-10T17:44:21+01:00",
+    "name": "Valheim Plus",
+    "author": "geoff.everaert@gmail.com",
+    "description": "A HarmonyX Mod aimed at improving the gameplay quality of Valheim. The mod includes several different main features including modifiers to ingame stats of players, buildings and entities and a sophisticated system to build and place objects with high precision and a system to modify already placed objects with high precision. The general goal is to provide V+ as a base modification for your gameplay to increase quality of life, change difficulty or have a better experience in general. The mod also comes with a version and configuration control system for servers and users, allowing servers to make sure that only people with the same configuration are able to join their servers.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
+    ],
+    "file_denylist": [],
+    "startup": ".\/start_server_plus.sh -name \"{{SERVER_NAME}}\" -port {{SERVER_PORT}} -world \"{{WORLD}}\" -password \"{{PASSWORD}}\" -public {{PUBLIC}} > >(sed -uE \"{{FILTER}}\") & trap \"{{STOP}}\" 15; wait $!",
```

Comment:
This line will not work with Filter ;)

Reply:
Will just remove and use it myself as it's probably gonna end up as a duplicate.